deepcell/image_generators/image_generators_test.py

```diff
@@ -38,6 +38,7 @@
 from tensorflow.python.platform import test
 
 from deepcell import image_generators
+from deepcell.image_generators import semantic
```

Comment:
We can remove this and import the `SemanticDataGenerator` directly from `image_generators` instead of importing `semantic`.